src/components/Button/Button.tsx

```diff
@@ -87,9 +87,25 @@ class Button extends UIComponent<any, any> {
     accessibility: ButtonBehavior as Accessibility,
   }
 
+  public renderIcon() {
+    const { content, icon, iconPosition, type } = this.props
+    const iconIsAfterButton = iconPosition === 'after'
+
+    const iconProps =
```

Comment:
Is this needed?
Can't you just call something like:
```JSX
Icon.create(this.props.icon, {
  defaultProps: { /* xSpacing and others */ },
})}
```
Shorthand factory can handle string inherently.

Reply:
yes, lets do this way, thanks!